### ISSUE TITLE
Click on a nested checkbox leads to an error

### ISSUE THREAD

**Issue by purelygrey** [careteditor/caret#218]
Example:
```
- Header
  - [  ] Checkbox 1
    - [  ] Checkbox 2
```

Click on the "Checkbox 2" in the preview window produces an error:
```
{
    "message": "Cannot read property 'nodes' of undefined",
    "version": "1.12.2",
    "os": "Windows 10",
    "path": "C:\\Users\\...\\AppData\\Local\\Caret\\app-1.12.2\\resources\\app.asar\\web\\Editor\\Editor.Markdown.js",
    "code": "_0x4cd0xe=_0x4cd0x2[_0xcf14[104]][0],_0x4cd0xf=this[_0xcf14[0]][_0xcf14[110]](_0x4cd0xe),_0x4cd0x10=_0x4cd0xf[_0xcf14[112]](_0xcf14[111])",
    "line": 5,
    "column": 16839,
    "stack": [
        "at Object.setTaskChecked ((C:\\Users\\...\\AppData\\Local\\Caret\\app-1.12.2\\resources\\app.asar\\web\\Editor\\Editor.Markdown.js:5:16839)",
        "at View._checkTaskList ((C:\\Users\\...\\AppData\\Local\\Caret\\app-1.12.2\\resources\\app.asar\\packages\\Preview\\Preview.js:5:6634)"
    ]
}
```

**Comment by astoilkov** [comment]
We will investigate. Thanks for reporting.

**Comment by purelygrey** [comment]
No problem! I also found an issue that looks similar but has less info: https://github.com/careteditor/caret/issues/187
Great editor by the way 👍

**Comment by astoilkov** [comment]
You are absolutely correct. Thanks for pointing out the issue.

**Comment by astoilkov** [comment]
This is now fixed in the latest Caret 1.13.0 release.

Closing this issue. Thanks for reporting.
